arm-sql/2015-05-01/swagger/sql.core.json

```diff
@@ -0,0 +1,184 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/move": {
```

Comment:
Since i think we're gonna need this definition again, I'm going to leave this and exclude the file in the composite.Json file

---
In reply to: [84132455](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r84132455) [](ancestors = 84132455,83115825,83113093,82919111,82916426)